api/v1/agent_upgrades.py

```
@@ -0,0 +1,543 @@
+            detail=f"Failed to get A/B test status: {str(e)}"
+        )
+
+
+# ============================================================================
+# ANALYTICS ENDPOINTS
+# ============================================================================
+
+@router.get("/analytics/summary")
+async def get_upgrade_analytics(
+    days: int = 30,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.5)

Unused function argument: `current_user`

(ARG001)